thinkhazard_processing/tests/common.py

```diff
@@ -0,0 +1,10 @@
+from thinkhazard_common.models import DBSession
+from ..models import Layer
+from sqlalchemy import func
+
+
+def new_geonode_id():
+    row = DBSession.query(func.max(Layer.geonode_id)).one_or_none()
+    if row[0] is None:
+        return 1
+    return row[0] + 1
```

Comment:
I think that @arnaud-morvan has added something similar to this already. To be checked.